Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/describe.go

```diff
@@ -602,6 +603,7 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 	return tabbedString(func(out io.Writer) error {
 		fmt.Fprintf(out, "Name:\t%s\n", pod.Name)
 		fmt.Fprintf(out, "Namespace:\t%s\n", pod.Namespace)
+		fmt.Fprintf(out, "SCC:\t%s\n", pod.Annotations[allocator.ValidatedSCCAnnotation])
```

Comment:
@smarterclayton - is that even possible to have an empty scc in openshift, doesn't every pod need to pass admission?  But, if possible, not sure what it would print and not sure best way to simulate that.  I could add to a new function similar to printControllers where if it's empty it would print "<none>" 

```
func printScc(pod *api.Pod) string {
	value, ok := pod.Annotations[allocator.ValidatedSCCAnnotation]
	if ok {
		return value
	}
	return "<none>"
}
```